src/sql/parts/common/customInputConverter.ts

```diff
@@ -129,6 +141,29 @@ function getQueryPlanEditorUri(input: EditorInput): URI {
 	return undefined;
 }
 
+
+/**
+ * If input is a supported notebook editor file (.ipynb), return it's URI. Otherwise return undefined.
+ * @param input The EditorInput to get the URI of
+ */
+function getNotebookEditorUri(input: EditorInput): URI {
+	if (!input || !input.getName()) {
+		return undefined;
+	}
+
+	// If this editor is not already of type notebook input
+	if (!(input instanceof NotebookInput)) {
```

Comment:
Have we tested the case where we re-convert for an existing notebook editor? What happens?